### DISCUSSION TITLE
build doesn't work

### DISCUSSION THREAD

**Discussion by AdrianPrendas** [payloadcms/payload#1603] [Q&A] [answered]
- The index.html file has admin/ in file path but the directory does not exist
- also how can I deploy it to production or change hostname to 0.0.0.0 ??

**Comment by jmikrut** [comment, marked as answer]
Hey @AdrianPrendas — we will need more information to help you here. 

In production, Payload serves the contents of the `/build` folder statically via Express, so everything under `/admin` will first attempt to match to a file in `/build`, but if there is no file there, it will fall back to serving to `/build/index.html`.

In regards to deploying to production, we have a collection of tutorials that show how to deploy Payload to a DigitalOcean droplet here:

https://payloadcms.com/docs/production/deployment#digitalocean-tutorials

But you can deploy it to anywhere that can host a Node / Express app.

Here's another walkthrough for Northflank:

https://northflank.com/guides/deploying-payload-cms

I'm going to convert this to a discussion as it is not related to an issue with Payload but we are happy to continue to help there in the discussion!
